### PR TITLE
devel/gearmand: update to 1.1.19.1

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: devel/gearmand/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # $NetBSD: Makefile,v 1.50 2020/06/02 08:23:36 adam Exp $
 
-DISTNAME=	gearmand-1.1.18
-PKGREVISION=	13
+DISTNAME=	gearmand-1.1.19
+PKGREVISION=	1
```
Comment (iamleot) [inline devel/gearmand/Makefile]: When updating package `PKGREVISION` should be removed.  Please remove it.

Comment (mmoll) [reply devel/gearmand/Makefile]: Thanks, removed.

**Comment by bsiegert** [comment]
Can you provide a commit message? That is, a changelog between the current version in pkgsrc and the new version. It can be copied verbatim from the source.

**Comment by mmoll** [comment]
> Can you provide a commit message?

I did update to 1.1.19.1 and now included the chnagelog in the git commit massage.

**Comment by bsiegert** [comment]
This fails to build for me (on NetBSD/aarch64) with the following error:

```
  CXXLD    bin/gearman
  CXXLD    t/unittest
ld: libtest/t_unittest-unittest.o: in function `lookup_false_TEST(void*)':
/usr/pkgsrc/devel/gearmand/work/gearmand-1.1.19.1/libtest/unittest.cc:973: warning: Warning: reference to the libc supplied alloca(3); this most likely will not work. Please use the compiler provided version of alloca(3), by supplying the appropriate compiler flags (e.g. not -std=c89).
ld: /usr/pkgsrc/devel/gearmand/work/gearmand-1.1.19.1/libtest/unittest.cc:973: undefined reference to `alloca'
ld: libtest/.libs/libtest.a(libtest_libtest_la-gearmand.o): in function `Gearmand::ping()':
/usr/pkgsrc/devel/gearmand/work/gearmand-1.1.19.1/libtest/gearmand.cc:86: undefined reference to `alloca'
gmake[2]: *** [Makefile:4842: t/unittest] Error 1
gmake[2]: Leaving directory '/usr/pkgsrc/devel/gearmand/work/gearmand-1.1.19.1'
```

Any ideas?

**Comment by mmoll** [comment]
> Any ideas?

Unfortunately not, does it build with 1.1.18?

**Comment by alarixnia** [comment]
@bsiegert It's using compiler flags that are incompatible with using the (non-standard) function alloca. Solved with:

~~~
BUILDLINK_TRANSFORM+=   opt:-std=c++0x:-std=gnu++0x
~~~

This can be closed.

**Comment by mmoll** [comment]
@niacat Thank You! 🙇🏿

**Pull request closed without merging**